docs/build.md

```diff
@@ -0,0 +1,25 @@
+# unofficial version of rmw_iceoryx
```

Comment:
If we merge this PR it would be the official version, right?

Reply:
Thanks, I will delete this md and write something in the changelog.